src/recensio/plone/browser/review.py

```diff
@@ -520,21 +519,14 @@ def getLicense(self):
                 if licence_obj:
                     licence_obj = licence_obj.to_object
                 if licence_obj:
-                    # XXX This needs to be rechecked, it seems that in the old
-                    # getTranslations() return a dict, butthe code was expected to be:
-                    # ```
-                    # licence_translated = licence_obj.getTranslation()
-                    # publication_licence = licence_translated.getText()
-                    # ```
-                    licence_translated = getTranslations(
-                        licence_obj, review_state=False
-                    )[""]
+                    licence_translated = licence_obj  # XXX get translation
+                    publication_licence = (
+                        licence_translated.text.output_relative_to(current) or ""
+                    )
                     licence_translated = licence_obj
                     publication_licence = licence_translated.text or ""
                     if publication_licence:
-                        publication_licence = publication_licence.output_relative_to(
-                            licence_translated
-                        )
+                        break
```

Comment:
This change is not valid, we need to rendere the rich text value

Reply:
That's done by line 524: `licence_translated.text.output_relative_to(current)`

Reply:
But `text` can be `None` that's why I added it after another `if`

Reply:
Like this? c81beb5

Reply:
No, that's wrong because at that point it can already be a text... Leave it like it was before please

Reply:
But you can't do `licence_translated.text or ""` and then call `output_relative_to` on that.

`AttributeError: 'str' object has no attribute 'output_relative_to'`

Also, you need to break when you've found something you want to use.

Reply:
Nope because you call output_relative_to if text is something...